### PR TITLE
Volunteers Page Beginnings

### PR DESCRIPTION
**Pull request by hanlinc27**
Recycling some of the old components for the beginning of the "Volunteers" Page
* Fixed the default uppercase on MaterialUI button
8c93ebcb-308b-4bee-bdab-e392559e1e6b

### CODE REVIEW

**Comment by BBoutet1** [comment]
@hanlinc27  TimelineSection is modified in this PR and your previous one too, both waiting for approval. This will not cause conflicts?

**Comment by hanlinc27** [comment]
@BBoutet1 I was hoping to merge in #98 and then rebase on that. :)

**Review by BBoutet1 — APPROVED** [review]

**Pull request merged**
